sellercenter-translation/src/main/java/com/gfg/sellercenter/translation/entity/Translation.java

```diff
@@ -0,0 +1,19 @@
+package com.gfg.sellercenter.translation.entity;
+
+import lombok.*;
+
+@Builder
+@Getter
+@Setter
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
+public class Translation {
+    private String namespace;
+
```

Comment:
What about to add @Column(name="NAMESPACE") if we are reading all the fields from sc database, the same for other table.

Reply:
This will never connect to the database. This "service"/package will call the graph service which will return the data which you need.